_topics/en/bip70-payment-protocol.md

```diff
@@ -0,0 +1,120 @@
+---
+title: BIP70 payment protocol
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Wallet Collaboration Tools
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **The BIP70 payment protocol** is an interactive protocol for
+  transmitting invoices and payments.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  The protocol involves several steps:
+
+  - Customer clicks a [BIP21][] `bitcoin:` URI extended with an
+    additional `r` parameter (described in [BIP72][]).  The URI handler
+    opens the user's wallet in response
+
+  - The wallet contacts the merchant's server and requests an invoice
+    signed by the merchant's SSL certificate.  Upon receipt and
+    validation of the signed invoice, the payment details are
+    automatically filled out on the wallet's Send Transaction screen.
+    Optionally, the user is notified that they're paying the owner of
+    the domain corresponding to signing certificate (e.g.
+    "example.com")
+
+  - The customer reviews the invoice details and clicks the *send
+    transaction* button.  The transaction is generated and broadcast to
+    the Bitcoin network.  A copy of the transaction is sent to the
+    merchant, who then replies with an acknowledgment that the payment
+    was received (optionally rebroadcasting the transaction from their
+    own nodes).
```

Comment:
We end the other bullets in this list without a . , suggestion to keep consistent